docs/conch/howto/conch_client.rst

```diff
@@ -181,7 +181,11 @@ For use of ``SSHCommandClientEndpoint`` that is intended to be completely autono
 
 It is also possible to run commands (one or more) over an
 already-established connection.  This is done using the alternate
-constructor ``SSHCommandClientEndpoint.existingConnection`` .
+constructor ``SSHCommandClientEndpoint.existingConnection`` .  The
+``connection`` argument to that function can be obtained from the
+the ``conn`` property of the ``transport`` property of the protocol
```

Comment:
(nit) this might be more simply stated as:
…by accessing `transport.conn` on the protocol…